content/TrueCommand/API/_index.md

```diff
@@ -1,11 +1,24 @@
 ---
-title: "API Interface"
-pre: "<i class='fa fa-bookmark'></i>	"
-geekdocCollapseSection: true
```

Comment:
the geekdocCollapseSection: true needs to be added back or the section is permanently expanded: 
![image](https://user-images.githubusercontent.com/17933320/118037053-58408d80-b33b-11eb-86ee-4b3c756a5562.png)

Reply:
I think we also need a _index.md file in the deprecated directory upstream, so those all get added together in a collapsible folder.  Those that I see that are randomly throw about are the ones in that directory.